GitCommands/Git/SubmoduleHelpers.cs

```diff
@@ -23,9 +23,9 @@ public static class SubmoduleHelpers
             return ParseSubmodulePatchStatus(patch, module, fileName);
         }
 
-        public static Task<GitSubmoduleStatus?> GetCurrentSubmoduleChangesAsync(GitModule module, string submodule, bool noLocks = false)
+        public static async Task<GitSubmoduleStatus?> GetCurrentSubmoduleChangesAsync(GitModule module, string submodule, bool noLocks = false)
```

Comment:
Why is this changing?

Reply:
The Async suffix indicated that the method should be async. I could revert this

Reply:
A method is asynchronous or not based on the use of a Task-like return type (e.g. `Task`, `Task<TResult>`, or `ValueTask<TResult>`). The `async` modifier has no impact on this determination.